infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -370,16 +370,17 @@ private CorfuRuntime getRuntime(ServerContext serverContext) {
             serverContext.getCorfuServerConnectionPort()));
 
         return CorfuRuntime.fromParameters(CorfuRuntime.CorfuRuntimeParameters.builder()
-            .trustStore((String) serverContext.getServerConfig().get(ConfigParamNames.TRUST_STORE))
-            .tsPasswordFile((String) serverContext.getServerConfig().get(ConfigParamNames.TRUST_STORE_PASS_FILE))
-            .keyStore((String) serverContext.getServerConfig().get(ConfigParamNames.KEY_STORE))
-            .ksPasswordFile((String) serverContext.getServerConfig().get(ConfigParamNames.KEY_STORE_PASS_FILE))
-            .tlsEnabled((Boolean) serverContext.getServerConfig().get("--enable-tls"))
-            .systemDownHandler(() -> System.exit(SYSTEM_EXIT_ERROR_CODE))
-            .maxCacheEntries(serverContext.getLogReplicationCacheMaxSize()/2)
-            .maxWriteSize(serverContext.getMaxWriteSize())
-            .build())
-            .parseConfigurationString(localCorfuEndpoint).connect();
+                .trustStore((String) serverContext.getServerConfig().get(ConfigParamNames.TRUST_STORE))
+                .tsPasswordFile((String) serverContext.getServerConfig().get(ConfigParamNames.TRUST_STORE_PASS_FILE))
+                .keyStore((String) serverContext.getServerConfig().get(ConfigParamNames.KEY_STORE))
+                .ksPasswordFile((String) serverContext.getServerConfig().get(ConfigParamNames.KEY_STORE_PASS_FILE))
+                .tlsEnabled((Boolean) serverContext.getServerConfig().get("--enable-tls"))
+                .systemDownHandler(() -> System.exit(SYSTEM_EXIT_ERROR_CODE))
+                .maxCacheEntries(serverContext.getLogReplicationCacheMaxSize()/2)
+                .maxWriteSize(serverContext.getMaxWriteSize())
+                .nullifyDataOnGetPayload(false)
```

Comment:
I agree with the renaming.

But moving this param to a config file will not help in resolving any bug that arises because of it. Reason being that even if this param is reverted to be true, LR will have a correctness issue where it won't be replicating some of the data...which in turn can cause more side effects like data loss.